.travis.yml

```diff
@@ -7,11 +7,11 @@ jdk:
 
 install:
   - export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
-  - mvn install -DskipTests
+  - mvn install -DskipTests -Dgpg.skip=true
```

Comment:
**I think users should be able to build the repo by just executing mvn clean install command.** For now as a temporary solution we can modify the readme file to  update the build command to mvn clean install -Dgpg.skip=true.

Reply:
I agree.  We can update it later to only sign when performing a release.  An example of how to do this is shown in this article: https://blog.sonatype.com/2010/01/how-to-generate-pgp-signatures-with-maven/